.gitignore

```diff
@@ -73,3 +74,6 @@ build/
 !**/ios/**/default.mode2v3
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
+
+# L10n files
+**/intl
```

Comment:
On this line, ignored the file lib/localization/intl/messages.dart.
Maybe need apply a regex to do not ignore this file and ignore the rest of them

Reply:
Aah, that one actually also needs to be removed. Done now.